pkgs/development/python-modules/barcodenumber/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python-stdnum
+}:
+buildPythonPackage rec {
+  pname = "barcodenumber";
+  version = "0.5.0";
+  format = "wheel";
+
+  # Bitbucket repo is no longer public or does no longer exist
```

Comment:
not a great sign, maybe a fork like https://gitlab.com/rubdos/python3-barcodenumber could help? didn't look into the details myself.